src/components/previewCard/PreviewCard.jsx

```diff
@@ -20,30 +21,33 @@ export const PreviewCard = ({ title, coverImg, description }) => {
         flexDirection: "column",
       }}
     >
-      <CardActionArea
-        sx={{ height: "100%", display: "flex", flexDirection: "column" }}
-      >
-        <CardMedia
-          component="img"
-          height="175"
-          image={coverImg}
-          alt="category image"
-        />
-        <CardContent sx={{ flexGrow: 1 }}>
-          <Typography
-            color="#fff"
-            fontWeight={600}
-            gutterBottom
-            variant="h5"
-            component="div"
-          >
-            {title}
-          </Typography>
-          <Typography color="#fff" variant="body1">
-            {description}
-          </Typography>
-        </CardContent>
-      </CardActionArea>
+        <CardActionArea
+          component={Link}
+          to="/explore"
+          onClick={onClick}
```

Comment:
better naming convention can be used for onClick function

Reply:
noted